spec/support/playbook/rspec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Playbook
+  module Rspec
+    extend RSpec::Matchers::DSL
+
+    matcher :define_prop do |prop_key|
+      chain :of_type do |type_class|
+        @type_class = type_class
+      end
+
+      chain :with_default do |default|
+        @default = default
+      end
```

Comment:
We might want to also add a helper for testing valid enum values

Reply:
Yep - just wrote you the same thing in a Connect message haha.

Reply:
Would be pretty easy to create one for each type.

`define_string_prop(:prop)`
`define_hash_prop(:prop)`
`define_boolean_prop(:prop)`
`define_enum_prop(:prop).with_values("foo", "bar").with_default("foo")`